app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
+
+    private static PebbleSync pebbleSync = new PebbleSync(mContext);
```

Comment:
but context wont exist at this point without the line above which we need to get rid of, but I see what you are saying now.

we need to change the pebble class a bit, Ill make some comments down there in a sec (basically when you are creating the pebble sync, it calls init, which has a receive data handler in it, which calls send data... seems like it would throw itself a few times then no?)
I feel like if we need that registered receiver, then the pebble sync should be a service all to itself, but then I am not super sure what the registerReceivedDataHandler is for or what it is triggering it (is that something we are triggering when we call sendData? or something the pebble itself triggers?)